src/test/ui/block-result/block-must-not-have-result-do.stderr

```diff
@@ -0,0 +1,14 @@
+error[E0308]: mismatched types
+  --> $DIR/block-must-not-have-result-do.rs:13:9
+   |
+11 | fn main() {
+   |           - expected `()` because of this default return type
```

Comment:
Why is the main function spanned on this? `loop`, `while` etc. can't return a value via tail expression